packages/tasit-action/src/ethers.test.js

```diff
@@ -60,27 +60,8 @@ describe("ethers.js", () => {
     expect(value).to.equal(rand);
   });
 
-  it("should set contract's value using signed tx", async () => {
-    var rand = Math.floor(Math.random() * Math.floor(1000)).toString();
-
-    const data = contract.interface.functions.setValue.encode([rand]);
-
-    const rawTx = {
-      gasLimit: 64000,
-      to: contract.address,
-      data: data,
-      nonce: await provider.getTransactionCount(wallet.address),
-    };
-
-    const signedTx = await wallet.sign(rawTx);
-    const sentTx = await provider.sendTransaction(signedTx);
-
-    await provider.waitForTransaction(sentTx.hash);
-
-    const value = await contract.getValue();
-    expect(value).to.equal(rand);
-  });
-
+  // Note: Non-Deterministisc test (It's failing sometimes).
+  // TODO: Using mineBlocks utils maybe could fix it.
```

Comment:
Yeah, I bet that would fix it

Reply:
That's no more failing after last https://github.com/tasitlabs/TasitSDK/pull/102 changes.

Reply:
That's great! Resolving this thread.